kotlin-inject-compiler/test/src/test/kotlin/me/tatarka/inject/test/FailureTest.kt

```diff
@@ -1495,7 +1495,7 @@ class FailureTest {
                 """.trimIndent()
             ).compile()
         }.output().all {
-            contains("e: [ksp] Cannot find an @Inject constructor or provider for: Foo")
+            contains("Cannot find an @Inject constructor or provider for: Foo")
```

Comment:
@evant The `e: [ksp]` prefix wasn't present anymore; it was a `toString` representation of `DiagnosticMessage`